l10n_it_account_stamp/migrations/18.0.1.0.0/post-migration.py

```diff
@@ -3,11 +3,13 @@
 
 from openupgradelib import openupgrade
 
+from odoo import SUPERUSER_ID, api
+
 
-@openupgrade.migrate()
```

Comment:
come mai serve togliere il decoratore? vedi thread a partire da https://github.com/OCA/l10n-italy/pull/4585#issuecomment-3024459389

Reply:
non dovrebbe, come ho evidenziato in https://github.com/OCA/l10n-italy/pull/4585#issuecomment-3024459389